tests/testthat/helper-testthat.R

```diff
@@ -1,3 +1,2 @@
-options(testthat.progress.verbose_skips = FALSE)
```

Comment:
This was removed in https://github.com/r-lib/testthat/commit/6cbd8190ac283a639ed7cff975f90f2c4d5db16d